### PR TITLE
Added freetube profile

### PR DESCRIPTION
**Pull request by kortewegdevries**
Request: https://github.com/netblue30/firejail/issues/1139#issue-213330086

What to do about the appimage? Do I delete it entirely? Tried with `?HAS_APPIMAGE: ignore ...` doesn't work.

### CODE REVIEW

File: etc/profile-a-l/freetube.profile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+include disable-interpreters.inc
+# include disable-shell.inc -Creates problems with --appimage
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/FreeTube
+whitelist ${HOME}/.config/FreeTube
+
+shell none
+ignore seccomp
+seccomp !chroot
```
Comment (rusty-snake) [inline etc/profile-a-l/freetube.profile:22]: :question: :question:

Comment (kortewegdevries) [reply etc/profile-a-l/freetube.profile:22]: electron profile has `seccomp` in it?

**Comment by kortewegdevries** [comment]
noblacklist bash,ignore shell etc.

File: etc/profile-a-l/freetube.profile (hunk 1 above)
Comment (rusty-snake) [reply etc/profile-a-l/freetube.profile:22]:
```
ignore seccomp
seccomp !chroot
```
=> no seccomp at all

```
seccomp !chroot
seccomp
```
=> seccomp with default-filter without chroot.

Comment (kortewegdevries) [reply etc/profile-a-l/freetube.profile:22]: Yeah but shouldn't it block all since electron profile has it?

Comment (rusty-snake) [reply etc/profile-a-l/freetube.profile:22]: https://github.com/netblue30/firejail/commit/3d8f587cd8e2604df928be21c4dd201bd0b818fc

**Comment by Fred-Barclay** [comment]
Squashed and merged, thanks!

**Pull request merged**